### DISCUSSION TITLE
[VOTE] Release Apollo version 2.0.1

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4395] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo version 2.0.1.

Release Note:

 * https://github.com/apolloconfig/apollo/releases/tag/v2.0.1

Release Candidate:

 * https://github.com/apolloconfig/apollo/releases/download/v2.0.1/apollo-configservice-2.0.1-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.0.1/apollo-adminservice-2.0.1-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.0.1/apollo-portal-2.0.1-github.zip
 * sha1 checksums
   - edec08cd6fa5c2281a9130427bfcc028ff58f614  apollo-configservice-2.0.1-github.zip
   - c3e860279e1406cf3c9b6230d3ec683f9a6d29b4  apollo-adminservice-2.0.1-github.zip
   - 872dee28ffafbec842fdc806b87c9419c016d55f  apollo-portal-2.0.1-github.zip

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1042/content/com/ctrip/framework/apollo/

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1042/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/apolloconfig/apollo/releases/tag/v2.0.1

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/e04028dfe9ca3e5792be89b96a6fbd2ee80355b7

Guide to build the release from source:

 * https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2212-building-from-source-code

Voting will start now (2022/6/4) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by Anilople** [comment]
+1 Release this package.

I've checked 
- [x] All unit tests passed.
- [x] Release Note is fine.
- [x] Download links are fine.
- [x] sha1 checksums are correct.

**Comment by lepdou** [comment]
+1 Release this package.

I've checked.

- [x] deploy 2.0.1 with standard distribute deployment.

**Comment by nisiyong** [comment]
+1 binding to release, and I have Checked
- [x] All uts passed.
- [x]  Release Note.
- [x]  Shasum are correct.
- [x] Download links are fine.
- [x] Deploying the new releases are fine.

**Comment by zouyx** [comment]
+1 binding to release

I've checked 
- [x] All uts passed.
- [x] Release Note is fine.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by nobodyiam** [comment]
The vote to release Apollo version 2.0.1 has passed with two +1 binding, two +1 and zero -1.
